source/features/git-checkout-pr.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import ClippyIcon from 'octicon/clippy.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {getCurrentBranch} from '../github-helpers';
+
+function handleMenuOpening({delegateTarget: dropdown}: delegate.Event): void {
+	dropdown.classList.add('rgh-git-checkout'); // Mark this as processed
+	const [, user, repository] = select<HTMLAnchorElement>('.commit-ref.head-ref a')!.pathname.split('/', 3);
+
+	const tabContainer = select('tab-container', dropdown)!;
+	tabContainer.style.width = '370px';
+	select('.UnderlineNav-body', tabContainer)!.append(
+		<button
+			name="type"
+			type="button"
+			role="tab"
+			aria-selected="false"
+			className="UnderlineNav-item flex-1 btn-link"
+			tabIndex={1}
+		>
+			Git Checkout
+		</button>
+	);
+
+	tabContainer.append(
+		<div hidden role="tabpanel" className="p-3" tabIndex={1}>
```

Comment:
This should be `0` just like the others
```suggestion
		<div hidden role="tabpanel" className="p-3" tabIndex={0}>
```